packages/xo-web/src/xo-app/jobs/schedules/index.js

```diff
@@ -209,12 +213,23 @@ export default class Schedules extends Component {
   ]
 
   render() {
-    const { cronPattern, jobs, schedule, schedules, timezone } = this.state
+    const {
+      cronPattern,
+      enabled,
+      jobs,
+      schedule,
+      schedules,
+      timezone,
+    } = this.state
     const userData = { jobs }
     return (
       <div>
         <h2>{_('newSchedule')}</h2>
         <form id='newScheduleForm'>
+          <div className='form-group'>
+            <label>{_('scheduleEnableAfterCreation')}</label>{' '}
```

Comment:
As discussed, this PR is dedicated to remove the condition that hide the posibility to enable/disable the schedule during the edition.

Reply:
My comment is about the displayed message not about the remove of the condition.

Should we display `Enable immediately after creation` on the schedule edition?

Reply:
yes, I understood your comment and that's why we have "decided" to only delete the condition and not to change the message.
We can differentiate the creation/edition in another PR because we also have the title "New schedule" which appears in the edition.